### ISSUE TITLE
업데이트 후 사용 불가

### ISSUE THREAD

**Issue by Sienna-lee** [edh-git/EL_Display_Hub#290]
- 문의 날짜 : 2022년 1월 20일 오후 2시 13분 
- 메일 주소 : dan_ali5@outlook.com

Title: Easy Canvas pro updates problem

Questions: Hi,

I updated easy canvas pro on my ipad and tried to update it on my PC I run the updates and it says, \"you already installed EL display Hub\" and it won\'t update. and now I cann\'t connect because of the different versions. don\'t know what to do to solve it.

Thank you,
Dan

[[EasyCanvas_-_Windows]_Easy_Canvas_pro_updates_problem.zip](https://github.com/edh-git/EL_Display_Hub/files/7902872/EasyCanvas_-_Windows._Easy_Canvas_pro_updates_problem.zip)

**Comment by james-kim-Devguru** [comment]
## 다음과 같이 답변 남겼습니다.
```
Hello, customer. Easy&Light support team.

First of all, we apologize for the inconvenience.
Currently, there are a lot of inquiries about the update, so the response is delayed.

EL Display Hub is an integrated PC program for Twomon SE, Easycanvas Easycanvas Pro.

It seems that the update on the application side does not work properly due to excessive EL Display Hub updates.
After updating the EL Display Hub, visit the App Store or Google Play Store on the tablet side and manually check the application update.

Also, it seems that the PC program is damaged in the process of updating the program on the current PC side. Reinstall the PC program as follows.

1. Uninstall both EL Display Hub and Easycanvas Pro on the PC side.
2. Reboot your PC.
3. Please reinstall EL Display Hub by installing a new installer.
[EL Display Hub] Installation page: http://www.easynlight.com/easycanvas/

If the problem is not improved even after taking the above measures, please contact us again.

thank you.

Easy & Light
```

**Comment by Sienna-lee** [comment]
해당 사용자가 아래와 같이 회신하였습니다.

----


Hi,

 

Thank you so much for your respond. I just want to let you know that it works now. I did the steps that you included in your email.

I uninstalled the old version app of easycanvas pro, then re-installed EL Display Hub only. And after reboot my pc, I connected my Ipad and PC with the wire (only the first time). Now everything is working using wifi connection. Thank you and appreciate your help.

 

Thank you,
D.A.
